change/@fluentui-web-components-290a7b72-4b1b-4b97-81f7-bad3330a0a6b.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28568&commitId=b14fb89f25a181f13bb7eccc2fec7a3ef782c610&env=prod&clientType=FLUENTUI)


  <details><summary><b>Button</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button.Small Icon Only - Dark Mode.pressed.chromium.png|413| Changed|
|Button.Small Icon Only.default.chromium.png|2405| Changed|
|Button.Small With Icon.default.chromium.png|2820| Changed|
|Button.Small With Icon - Dark Mode.hover.chromium.png|946| Changed|
|Button.Small With Icon - Dark Mode.default.chromium.png|887| Changed|
|Button.Small With Icon - RTL.hover.chromium.png|2745| Changed|
|Button.Small With Icon - Dark Mode.pressed.chromium.png|875| Changed|
|Button.Small.hover.chromium.png|2420| Changed|
|Button.Small With Icon.pressed.chromium.png|2590| Changed|
|Button.Small Icon Only - Dark Mode.default.chromium.png|414| Changed|
|Button.Small With Icon - RTL.default.chromium.png|2819| Changed|
|Button.Small.default.chromium.png|2444| Changed|
|Button.Small Icon Only.hover.chromium.png|2395| Changed|
|Button.Small.pressed.chromium.png|2369| Changed|
|Button.Small Icon Only - Dark Mode.hover.chromium.png|422| Changed|
|Button.Small With Icon - RTL.pressed.chromium.png|2590| Changed|
|Button.Small With Icon.hover.chromium.png|2745| Changed|
|Button.Small Icon Only.pressed.chromium.png|2374| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>